utils/run_test.sh

```diff
@@ -52,6 +52,26 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES="${DAOS_BASE}"/test_results/*.xml
```

Comment:
(lint) Brace expansions and globs are literal in assignments. Quote it or use an array. [SC2125]